pallets/xst/benchmarking/src/lib.rs

```diff
@@ -0,0 +1,207 @@
+// This file is part of the SORA network and Polkaswap app.
```

Comment:
Why do you need to move XST benchmarking in separate folder?

Reply:
We have changes for these benchmarks in #374 please don't miss them.
@vovac12 fyi if Nikita commit in develop before #374 you need to merge it

Reply:
@wer1st this is due to usage of band and oracle-proxy pallets. If we put them into dependencies section, they will cause cyclic dependency error afaik